theano/tensor/elemwise.py

```diff
@@ -477,14 +476,16 @@ class Elemwise(OpenMPOp):
 
     """
 
+    __props__ = ("scalar_op", "inplace_pattern", "name", "nfunc_spec", "openmp")
```

Comment:
Normally, name should not be a props. I think we should remove it at the same time. We removed it from another node that had it. We don't do this anymore, this is old stuff. Remove it instead from props and from \__init__.